manifests/config_cmd.pp

```diff
@@ -30,21 +34,22 @@
     $base_cmd = "qpid-config -b amqp://${hostname}:${_port}"
   }
 
+  Service <| tag == 'qpidd' |> ->
```

Comment:
That is correct. It does look like the exact syntax I used (chain to an in with resources inside it) is puppet 4 only so I'll modify it to work on puppet 3 as well. You can see it in the tests as well where I test both with an without qpid.